### PR TITLE
dokku-deploy: add action

### PR DESCRIPTION
**Pull request by lastoCHka42**
Added action to deploy apps to a Dokku remote server. Based on idoberko2/dokku-deploy-github-action.

### CODE REVIEW

File: dokku-deploy-github-action/action.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    default: "master"
+  git-push-flags:
+    description: "Additional flags to be passed to the git push command. Could be used to force push"
+    required: false
+    default: ""
+  ssh-port:
+    description: "Custom port for ssh host"
+    required: false
+    default: "22"
+  commit:
```
Comment (ylobankov) [inline dokku-deploy-github-action/action.yml:33]: Shouldn't it be named `git-ref`?

File: dokku-deploy-github-action/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+It is recommended to pass `actions/checkout` the `fetch-depth: 0` parameter to avoid errors such as `shallow update not allowed`
+
+```
+steps:
+  - uses: actions/checkout@v2
+    with:
+        fetch-depth: 0
+  - id: deploy
+    name: Deploy to dokku
+    uses: idoberko2/dokku-deploy-github-action@v1
```
[the remark this answers is not in the document]
Comment (lastoCHka42) [reply dokku-deploy-github-action/README.md]: fixed

File: dokku-deploy-github-action/action.yml (hunk 1 above)
Comment (lastoCHka42) [reply dokku-deploy-github-action/action.yml]: fixed

**Review by ochaplashkin — APPROVED** [review]
LGTM :rocket:

File: dokku-deploy/README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+# dokku-deploy
+
+This action makes deployment to Dokku as easy as possible!
+Based on [idoberko2/dokku-deploy-github-action](https://github.com/idoberko2/dokku-deploy-github-action).
+
+## Inputs
+
+### ssh-private-key
```
Comment (ylobankov) [inline dokku-deploy/README.md:8]: I think it is better to specify that is a required field here.

```
ssh-private-key (required)
```

File: dokku-deploy/README.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+### ssh-port
+
+Port for ssh host. If not specified, `22` will be used.
+
+### dokku-user
+
+The user to login into the Dokku host. If not specified, "dokku" user will be used.
+
+### dokku-host
```
Comment (ylobankov) [inline dokku-deploy/README.md:26]: ditto.

File: dokku-deploy/README.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+The Dokku host to deploy to.
+
+### dokku-app-name
```
Comment (ylobankov) [inline dokku-deploy/README.md:30]: ditto.

**Review by ylobankov — APPROVED** [review]

File: dokku-deploy/README.md (hunk 3 above)
Comment (lastoCHka42) [reply dokku-deploy/README.md]: done

File: dokku-deploy/README.md (hunk 4 above)
Comment (lastoCHka42) [reply dokku-deploy/README.md]: done

File: dokku-deploy/README.md (hunk 5 above)
Comment (lastoCHka42) [reply dokku-deploy/README.md]: done

File: dokku-deploy/README.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+        fetch-depth: 0
+  - id: deploy
+    name: Deploy to Dokku
+    uses: tarantool/actions/dokku-deploy@master 
+    with:
+        ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
+        dokku-host: 'my-dokku-host.com'
+        dokku-app-name: 'my-dokku-app'
+```
+
```
Comment (ylobankov) [inline dokku-deploy/README.md:75]: Please remove empty line.

File: dokku-deploy/README.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+hint: to the same ref. You may want to first integrate the remote changes
+hint: (e.g., 'git pull ...') before pushing again.
+hint: See the 'Note about fast-forwards' in 'git push --help' for details.
+```
+
+### git-ref
+
+Branch, tag or commit SHA to deploy. If not specified, `$GITHUB_SHA` (current commit) will be used.
+
+
```
Comment (ylobankov) [inline dokku-deploy/README.md:54]: Please remove empty line.

File: dokku-deploy/action.yml [hunk 8]
```diff
@@ -0,0 +1,44 @@
+name: Dokku deploy action
+description: Deploy to a Dokku host using GitHub Actions
+inputs:
+  ssh-private-key:
+    description: >
+      The ssh private key to the Dokku host. WARNING: use a secret! 
+      Do not pass a plain text value!
+    required: true
+  ssh-port:
+    description: Custom port for ssh host
```
Comment (ylobankov) [inline dokku-deploy/action.yml:10]: Why does this description differ from the description in README.md? Could you please check all descriptions in this file and make them consistent?

File: dokku-deploy/README.md (hunk 7 above)
Comment (lastoCHka42) [reply dokku-deploy/README.md]: done

File: dokku-deploy/action.yml (hunk 8 above)
Comment (lastoCHka42) [reply dokku-deploy/action.yml]: fixed

**Review by ylobankov — APPROVED** [review]

File: dokku-deploy/README.md (hunk 6 above)
Comment (lastoCHka42) [reply dokku-deploy/README.md]: done

**Pull request merged**